RubyTask1.rb

```diff
@@ -0,0 +1,334 @@
+
+ 
+  require 'prime'
+
+  ###################################################################
+  #================================================================#
+           # HELLO This is RUBY Task One #
+  #=================================================================#
+  ###################################################################
+
+
+  # Array ExcercisesArray1 = [1, 3, 4, 4, 1, 2, 3, 12, 19, 18, 2, 10]
+  # Array2 = [2, 3, 5, 8, 9, 12, 18, 15, 1, 24, 21]
+  # Hash = {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6}
+  # Nested Array = [1, [1, 2, [1, 2, 3, [1, 2, 3, 4]]]]
+  # 1. sort in ascending order\
+  # 2. sort in descending order
+  # 3. find duplicates
+  # 4. find prime numbers
+  # 5. intersect arrays 
+  # 6. union arrays
+  # 7. find 8th element from Array1
+  # 8. find 12 value's index number
+  # 9. add 3, 4, 9, 12 in Array2
+  # 10. delete 18, 19 from Array1
+  # 11. format hash as Array1 is key and Array 2 as values like {1 => 2, 3 => 3, 4 => 5, 4 => 8 etc...}
+  # 12. sort Hash by key (asc, desc)
+  # 13. sort Hash by value (asc, desc)
+  # 14. add {g: 7} in Hash
+  # 15. remove {a: 1} from Hash
+  # 16. convert nested array to same level and remove duplicates as [1, 2,3,4]
+
+  
+  ###########################################################################
+    #   Start Coding..  #
+   
+   ###########
+   #Array ONE# ( Ascending  & Descending )
+   ###########
+   arrayOne = Array.new()
```

Comment:
`arrayOne` is not correct variable name
it should be
`array_one`

To initialize an Array
1. `array_one= Array.new([1,3,4,4,5,5,5,1,5,5,5,])`
2. post preferred way is `array_one = [1,3,4,4,5,5,5,1,5,5,5]`